scripts/generate_metadata.spec.js

```diff
@@ -0,0 +1,86 @@
+import { generateMetadata, writeFile } from "./generate_metadata";
```

Comment:
Please add a test that covers a HTTP error and please assert against the exit code.